src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -1061,12 +936,8 @@ else if (!authCheckService.isAtLeastTeachingAssistantForExercise(programmingExer
     @FeatureToggle(Feature.PROGRAMMING_EXERCISES)
     public ResponseEntity<Resource> exportSubmissionsByParticipationIds(@PathVariable long exerciseId, @PathVariable String participationIds,
             @RequestBody RepositoryExportOptionsDTO repositoryExportOptions) throws IOException {
-        ProgrammingExercise programmingExercise = programmingExerciseRepository.findByIdWithStudentParticipationsAndSubmissionsElseThrow(exerciseId);
-
-        if (!authCheckService.isAtLeastTeachingAssistantForExercise(programmingExercise)) {
-            return forbidden();
-        }
-
+        var programmingExercise = programmingExerciseRepository.findByIdWithStudentParticipationsAndSubmissionsElseThrow(exerciseId);
+        authCheckService.checkIsAtLeastInstructorForExerciseElseThrow(programmingExercise, null);
```

Comment:
This was `isAtLeastTeachingAssistantForExercise` before. Is this intended? If it is, we should remove the `'TA'` from `hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')`

Reply:
(This was the only line that I could find where this changed)

Reply:
probably a mistake which was not identified by the tests. good catch

Reply:
fixed